blueprints/getting-started/.gitignore

```diff
@@ -3,3 +3,4 @@
 .terraform.lock.hcl
 *.tfstate
 *.tfstate.*
+kubeconfig_*
```

Comment:
EKS operations _may_ generate a `kubeconfig` file. This should be ignored and not added to VCS.